### PR TITLE
Add MongoDB support

### PR DESCRIPTION
**Pull request by gchaix**
Per https://docs.mongodb.com/manual/tutorial/install-mongodb-on-red-hat/

### CODE REVIEW

File: manifests/mongodb.pp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  file { '/etc/pki/rpm-gpg/RPM-GPG-KEY-MongoDB':
+    ensure => present,
+    owner  => root,
+    group  => root,
+    mode   => '0644',
+    source => 'puppet:///modules/yumrepos/etc/pki/rpm-gpg/RPM-GPG-KEY-MongoDB',
+  }
+
+  yumrepo { 'mongodb':
+    descr       => 'MongoDB0',
```
Comment (jeffsheltren) [inline manifests/mongodb.pp]: Why 0 at the end?

File: manifests/params.pp [hunk 2]
```diff
@@ -160,6 +160,13 @@
   $mariadb10_includepkgs = absent
   $mariadb10_exclude = absent
 
+  # MongoDB
+  $mongodb_url = "https://repo.mongodb.org/yum/redhat/${::operatingsystemmajrelease}/mongodb-org/3.2/x86_64/"
```
Comment (jeffsheltren) [inline manifests/params.pp]: Change x86_64 to `${::architecture}` just for fun...

File: manifests/mongodb.pp (hunk 1 above)
Comment (gchaix) [inline manifests/mongodb.pp]: Typo.

File: manifests/params.pp (hunk 2 above)
Comment (gchaix) [inline manifests/params.pp]: The Mongo docs say they only support x86_64, but I don't see any harm in doing it just for fun.

**Pull request merged**